app/controllers/application_controller.rb

```diff
@@ -1,7 +1,17 @@
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
   before_action :require_signin_permission!
 
-  protect_from_forgery with: :exception
+  rescue_from ActionController::ParameterMissing do |e|
+    render json: { error: { message: e.message } }, status: 400
+  end
```

Comment:
We still have to include the `rescue_from` method otherwise the `ParameterMissing` exception gets passed all the way to the test. Plus we get the features of strong parameters in Rails 5 now automatically, by using `fetch` we get that exact `ParameterMissing` exception.